backend/functions/lib/use-cases/offices/offices.ts

```diff
@@ -17,8 +17,23 @@ const MODULE_NAME = 'OFFICES_USE_CASE';
 
 export class OfficesUseCase {
   public async getOffices(context: ApplicationContext): Promise<UstpOfficeDetails[]> {
-    const gateway = getOfficesGateway(context);
-    return gateway.getOffices(context);
+    const officesGateway = getOfficesGateway(context);
+    const offices = await officesGateway.getOffices(context);
+
+    const storageGateway = getStorageGateway(context);
+    const metas = storageGateway.getUstpDivisionMeta();
+
+    offices.forEach((ustpOffice) => {
+      ustpOffice.groups.forEach((group) => {
+        group.divisions.forEach((division) => {
+          if (metas.has(division.divisionCode)) {
+            division.isLegacy = metas.get(division.divisionCode).isLegacy;
```

Comment:
With the current definition of UstpDivisionMeta, this assignment seems to violate the type:
```
export type UstpDivisionMeta = {
  isLegacy?: true;
};
```
So if `isLegacy` exists, it should be `true`, but we're potentially setting it to `false`.

Reply:
I intend for it to be absent if not applicable.